dev/AnimatedIcon/AnimatedIcon.idl

```diff
@@ -3,26 +3,26 @@
 
 [WUXC_VERSION_PREVIEW]
 [webhosthidden]
-interface IRichAnimatedVisualSource
+interface IAnimatedVisualSource2
 {
-    IAnimatedVisual TryCreateAnimatedIconVisual(Windows.UI.Composition.Compositor compositor);
+    IAnimatedVisual TryCreateAnimatedVisual(Windows.UI.Composition.Compositor compositor);
     Windows.Foundation.Collections.IMapView<String, Double> Markers { get; };
     void SetColorProperty(String propertyName, Windows.UI.Color value);
 };
 
 [WUXC_VERSION_PREVIEW]
```

Comment:
>WUXC_VERSION_PREVIEW [](start = 1, length = 20)

Need to merge master: This is now MUX_PREVIEW. But is AnimatedIcon signed off yet?

Reply:
The review meeting is tomorrow.